src/style.css

```
@@ -1 +1,85 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  margin: 20px auto 0;
+  width: 100%;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.logo__image {
+  margin-left: 50px;
+  padding: 0;
+  height: 40px;
+  width: 40px;
+}
+
+nav .nav__list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  gap: 20px;
+  margin-right: 50px;
+}
+
+.nav__item {
+  padding-top: 5px;
+  padding-bottom: 5px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.nav__link {
+  font-weight: 500px;
+  font-size: 12px;
+  list-style-type: none;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  text-decoration: underline;
```

there should be no underline
```suggestion
  text-transform: uppercase;
  text-decoration: none;
}

.nav__link:hover {
```